docs-2.0/3.ngql-guide/3.data-types/9.type-conversion.md

```diff
@@ -0,0 +1,24 @@
+# Type Conversion
+
+Converting an expression of a given type to another type is known as type-conversion. In nGQL, type conversion is divided into explicit conversion.
```

Comment:
known as type conversion. No "-" is needed. Check the rest of the topic for the same problem.